packages/experiments/src/utilities/BaseState.tsx

```diff
@@ -27,20 +32,16 @@ export class BaseState<TProps, TState> extends React.Component<TProps, TState> {
   }
 
   public render(): JSX.Element | null {
-    // tslint:disable-next-line:no-any
-    const { renderView, ...rest } = this.props as any;
-
-    const newProps: TProps = {
-      ...rest,
```

Comment:
@dzearing I took `...rest` out since createComponent will handle that. Just wanted to call it out in case you disagree.

Reply:
i think it's ok.